tests/functional/api/test_epics.py

```diff
@@ -32,3 +37,42 @@ def test_epic_notes(epic):
     epic.notes.create({"body": "Test note"})
     new_notes = epic.notes.list(get_all=True)
     assert len(new_notes) == (len(notes) + 1), f"{new_notes} {notes}"
+
+
+def test_epic_save_from_parent_group_updates_subgroup_epic(gl, group):
+    subgroup_id = uuid.uuid4().hex
+    subgroup = gl.groups.create(
+        {
+            "name": f"subgroup-{subgroup_id}",
+            "path": f"sg-{subgroup_id}",
+            "parent_id": group.id,
+        }
+    )
+
+    nested_epic = subgroup.epics.create(
```

Comment:
Could we have this one as a fixture that `yield`s the epic, to avoid the manual cleanup below?

Or I'm just confused by the try/finally block, maybe it was meant to test deletion even if previous things failed.